packages/mdl-drawer/temporary/index.js

```diff
@@ -57,6 +57,8 @@ export default class MDLTemporaryDrawer extends MDLComponent {
           this.drawer.removeEventListener(util.remapEvent(evt), handler),
       registerTransitionEndHandler: handler => this.drawer.addEventListener('transitionend', handler),
       deregisterTransitionEndHandler: handler => this.drawer.removeEventListener('transitionend', handler),
+      registerDocumentKeydownHandler: handler => document.addEventListener('keydown', handler),
```

Comment:
Yes, I agree. I'm ok with merging this as-is and refactoring later, once we have focus-trapping code in a shared util, but I'll let you decide.